.ci/Jenkinsfile

```diff
@@ -375,6 +379,12 @@ def withCloudTestEnv(Closure body) {
     [var: "AWS_ACCESS_KEY_ID", password: aws.access_key],
     [var: "AWS_SECRET_ACCESS_KEY", password: aws.secret_key],
   ])
+  // GCP
+  withGCPEnv(secret: env.ELASTIC_PACKAGE_GCP_SECRET) {
+    maskedVars.add([var: 'GOOGLE_CREDENTIALS', password: readFile(file: env.GOOGLE_APPLICATION_CREDENTIALS)]);
+    maskedVars.add([var: 'GCP_PROJECT_ID', password: env.ELASTIC_OBSERVABILITY_PROJECT_ID])
+  }
+  // Masking
```

Comment:
by doing so, aren't you adding this for all the parallel stages regardless whether they interact with GCP or no? I wonder whether this should be done only when needed, and it's feasible?

Reply:
Yes, that's a side effect of refactoring done some time ago. Most likely it can be improved, but let's keep it aside from this PR. It's consistent with AWS at the moment.